helm/discovery_service/apis/beforeHandler.js

```diff
@@ -50,12 +50,17 @@
 
 
 */
+const { ExecutionContext } = require('../packages/discovery/lib/core');
+module.exports = function (req, finished) {
 
-module.exports = function(req, finished) {
 
-  console.log('beforeHandler in discovery_service invoked!');
+	console.log('beforeHandler in discovery_service invoked!');
 
-  req.qewdSession = this.qewdSessionByJWT.call(this, req);
-  return true;
+	req.qewdSession = this.qewdSessionByJWT.call(this, req);
+	const authorised = this.jwt.handlers.validateRestRequest.call(this, req, finished);
```

Comment:
@DmitrySolyannik `validateRestRequest` is not needed here. Please refer to `beforeHandler` logic [here](https://github.com/robtweed/qewd/blob/master/up/ms_handlers.js#L215)

Reply:
Also, seems formatting is not valid, please check all files for using 2 spaces

Reply:
Alexey is correct - QEWD-Up checks the authorisation of all requests automatically unless you specifically tell it not to - via the optional authenticate: false property for a route.  So what you're doing is not needed and should be removed  - my original beforeHandler.js logic was correct and should be left unchanged

Reply:
For more info see: https://github.com/robtweed/qewd/blob/master/up/docs/Routes.md#authenticated-jwts-in-microservice-applications